lobster_simulator/tools/PybulletAPI.py

```diff
@@ -195,10 +193,6 @@ def rayTest(rayFromPosition: Vec3, rayToPosition: Vec3) -> Tuple[float, Vec3, Ve
 
         return hit_fraction, Vec3.fromENU(hit_position), Vec3.fromENU(hit_normal)
 
-    # @staticmethod
-    # def createMultiBody()
-
-    @staticmethod
     def applyExternalTorque(objectUniqueId: int, torqueObj: Vec3, frame: Frame):
```

Comment:
If you remove the @staticmethod you should add the self parameter to the method. Didn't test it? ;)
```suggestion
    def applyExternalTorque(self, objectUniqueId: int, torqueObj: Vec3, frame: Frame):
```

Reply:
Wait, why is the static method actually removed? I don't it should right?

Reply:
Thanks for fixing it